src/components/header/IntervalMarkerOverview.js

```diff
@@ -107,7 +104,7 @@ class IntervalMarkerOverview extends PureComponent {
     return null;
   }
 
-  _onMouseMove(event: SyntheticMouseEvent) {
+  _onMouseMove(event: SyntheticMouseEvent<>) {
```

Comment:
What's the difference between `<>` and `<*>`? I feel like I've always done the latter, but never the former.

Reply:
In this case, it's easy as the generic has a default value, see https://github.com/facebook/flow/blob/c803f7abe609a896a0f54e3654e99348a9f50370/lib/react-dom.js#L52. So if you put `<>` without any specified value it will use the default value `EventTarget`.

But I honestly don't know what's the behavior is supposed to be when there's no default value like in https://github.com/devtools-html/perf.html/pull/622/files#diff-7075994d07133c249615a64660cfab09R40. Maybe it uses `any` for what I know. For that one I should likely find a value that prevents using the IDBKeyRange properties from the window property, maybe "empty" would work...

When you use `*` you just asks Flow to infer the value from the environment. I see it like the `_` in Rust (in `let vec: Vec<_> = XXX`). So in the end you should get a proper type, and Flow should still check it.